city_scrapers/spiders/chi_state_university.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiStateUniversitySpider(CityScrapersSpider):
+    name = "chi_state_university"
+    agency = "Chicago State University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.csu.edu/boardoftrustees/dates.htm"]
+
+    def parse(self, response):
+        for item in response.css(".table tr td li"):
```

Comment:
We're going to want to pull committee meetings and special board meetings separately with different titles

Reply:
How can I split in this case?

Reply:
You can start from each `p` tag, check for the `span` with the `smallgreenTitle` class and then pull the meeting items with `following-sibling` in XPath

Reply:
Which will be the `parent element`? the `p` tag?

Reply:
Not sure I'm following, but I think so